lib/eibflatpak.py

```diff
@@ -344,17 +351,36 @@ def reset_metadata(self):
             remote.set_title(self.title)
         if self.default_branch:
             remote.set_default_branch(self.default_branch)
+        if self.prio is not None:
+            remote.set_prio(self.prio)
+        if self.noenumerate is not None:
+            remote.set_noenumerate(self.noenumerate)
```

Comment:
I think you'd also need to include `self.prio` and `self.noenumerate` into the early return at the beginning of this method.